crates/core/src/db/datastore/locking_tx_datastore/mod.rs

```diff
@@ -669,7 +823,7 @@ impl Inner {
         Err(SequenceError::UnableToAllocate(seq_id).into())
     }
 
-    fn create_sequence(&mut self, seq: SequenceDef) -> super::Result<SequenceId> {
+    fn create_sequence(&mut self, seq: SequenceDef, database_address: Address) -> super::Result<SequenceId> {
```

Comment:
For next time: I think we could have factored this PR in two. Some of these changes are very mechanical, and IMO could have landed separately first. I'm always going to ask for more and smaller PRs.